packages/opentelemetry-plugin-ioredis/src/ioredis.ts

```diff
@@ -0,0 +1,59 @@
+/*!
+ * Copyright 2019, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { BasePlugin } from '@opentelemetry/core';
+import * as ioredisTypes from 'ioredis';
+import * as shimmer from 'shimmer';
+import { getTracedSendCommand } from './utils';
+
+export class IORedisPlugin extends BasePlugin<typeof ioredisTypes> {
+  static readonly COMPONENT = 'ioredis';
+  readonly supportedVersions = ['^4.14.0'];
```

Comment:
I believe this instrumention should works for version `>2`, did you encounter an issue with older versions ?

Reply:
True!